packages/proxy/lib/http/response-middleware.ts

```diff
@@ -225,6 +239,23 @@ const SetInjectionLevel: ResponseMiddleware = function () {
   this.next()
 }
 
+// https://github.com/cypress-io/cypress/issues/6480
+const MaybeStripDocumentDomainFeaturePolicy: ResponseMiddleware = function () {
+  const { 'feature-policy': featurePolicy } = this.incomingRes.headers
+
+  if (featurePolicy) {
+    const directives = parseFeaturePolicy(<string>featurePolicy)
+
+    if (directives['document-domain']) {
+      delete directives['document-domain']
+
+      this.res.set('feature-policy', stringifyFeaturePolicy(directives))
```

Comment:
If the user passes `Feature-Policy: document-domain 'none'`, this will cause `Feature-Policy:` to be set instead. I believe it's invalid to set an empty HTTP header, so can this be updated to just completely remove the header if `stringifyFeaturePolicy(directives)` is empty?

Reply:
@flotwig will do! About adding a test for this scenario: will a unit test suffice or do you want me to write an integration test as well?

Reply:
Unit would be fine

Reply:
@flotwig all done, you can check again whenever you have time :)